CMP1903_A1_2324/Program.cs

```diff
@@ -6,14 +6,23 @@
 
 namespace CMP1903_A1_2324
 {
-    internal class Program
+    public class Program
     {
-        static void Main(string[] args)
+        public static void Main(string[] args)
         {
-            /*
-             * Create a Game object and call its methods.
-             * Create a Testing object to verify the output and operation of the other classes.
-             */
+            Program.Run();
+        }
+
+        public static void Run()
+        {
+            // Creating a game object
+            Game game = new Game();
+            // Rolling the dice for the turn
+            int total = game.RollAllTheDice();
+            // Will perform all the testing in place
+            Testing.Test();
+            Console.ReadLine();
```

Comment:
This runs the program as the requirement states, with no problems.
You also managed to use the objects from other part of the program in your main program, Well done.
You are also making sure the testing is done while the program runs at the same time.